src/pages/login/Login.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
+import './Login.scss';
```

Comment:
```suggestion
import React, {useState} from 'react';
import { useNavigate } from 'react-router-dom';
import './Login.scss';
```
이렇게 하나로 구현해 줄 수 있겠습니다

Reply:
훨씬 깔끔하고 보기 좋은것 같습니다! 수정 했습니다 감사합니다!